Makefile

```diff
@@ -8,25 +8,22 @@ DESTDIR ?=
 EPOCH_TEST_COMMIT ?= $(shell git merge-base $${DEST_BRANCH:-main} HEAD)
 HEAD ?= HEAD
 
-export PODMAN_VERSION ?= "3.2.0"
+export PODMAN_VERSION ?= "4.0.0"
 
 .PHONY: podman
 podman:
 	rm dist/* || :
-	python -m pip install --user -r requirements.txt
+	$(PYTHON) -m pip install --user -r requirements.txt
 	PODMAN_VERSION=$(PODMAN_VERSION) \
 	$(PYTHON) setup.py sdist bdist bdist_wheel
 
 .PHONY: lint
-lint:
-	$(PYTHON) -m pylint podman || exit $$(($$? % 4));
+lint: tox
+	$(PYTHON) -m tox -e black,pylint
 
 .PHONY: tests
-tests:
-	python -m pip install --user -r test-requirements.txt
-	DEBUG=1 coverage run -m unittest discover -s podman/tests
-	coverage report -m --skip-covered --fail-under=80 --omit=./podman/tests/* --omit=.tox/* \
-	--omit=/usr/lib/* --omit=*/lib/python*
+tests: tox
+	DEBUG=1 $(PYTHON) -m tox -e pylint,coverage,py36,py38,py39,py310
```

Comment:
coverage should run the tests so you probably don't need to add py3* in this list

Reply:
@mwhahaha When I remove the py*, tox only runs for the default interpreter rather than iterating over the versions.

Reply:
Right but it was failing in CI because some of them weren't installed.

Reply:
@mwhahaha Yup, @cevich is building a new image with all the interpreters (as I type) and changes in the podman build tool chain is also tripping me up. Will get there. :+1: